GsaGH/Components/4_Analysis/SteelSectionPoolNames.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Drawing;
+using Grasshopper;
+using Grasshopper.Kernel;
+using Grasshopper.Kernel.Data;
+using Grasshopper.Kernel.Types;
+using GsaGH.Helpers.GH;
+using GsaGH.Parameters;
+using GsaGH.Properties;
+using OasysGH;
+using OasysGH.Components;
+
+namespace GsaGH.Components {
+  public class SteelSectionPoolNames : GH_OasysComponent {
+    public override Guid ComponentGuid => new Guid("6df7c85e-f629-47fd-ab38-7431407777e3");
+    public override GH_Exposure Exposure => GH_Exposure.quinary | GH_Exposure.obscure;
+    public override OasysPluginInfo PluginInfo => GsaGH.PluginInfo.Instance;
+    protected override Bitmap Icon => Resources.SteelSectionPoolNames;
+
+    public SteelSectionPoolNames() : base("Steel Section Pool Names", "Pool Names",
+      "Get or set the Steel Section Pool Names of a GSA Model", CategoryName.Name(),
+      SubCategoryName.Cat4()) {
```

Comment:
Should this not be in the Model subcategory?

Reply:
Where do you wanna put it exactly?

Reply:
Thinking either of these two:
![image](https://github.com/arup-group/GSA-Grasshopper/assets/25223248/3b690f4a-bdec-405e-bc20-1d6a7df3b96d)